test/serializer_test.exs

```diff
@@ -55,8 +55,8 @@ defmodule Membrane.Element.IVF.SerializerTest do
   correctly calculates timestamp in frame header.
   """
   test "appends headers correctly" do
-    buffer_1 = %Buffer{payload: @frame, metadata: %{timestamp: 0}}
-    buffer_2 = %Buffer{payload: @frame, metadata: %{timestamp: 100_000_000 <|> 3}}
+    buffer_1 = %Buffer{payload: @frame, pts: 0}
+    buffer_2 = %Buffer{payload: @frame, pts: 100_000_000 <|> 3}
```

Comment:
we no longer accept ratio as the timestamp

Reply:
I have created issue for this - [MV-71](https://membraneframework.atlassian.net/browse/MV-71)